tool/net/redbean.c

```diff
@@ -3514,7 +3518,8 @@ static void LuaSetIntField(lua_State *L, const char *k, lua_Integer v) {
 }
 
 static int LuaLaunchBrowser(lua_State *L) {
-  LaunchBrowser();
+  const char *p = luaL_checkstring(L, 1);
```

Comment:
Consider `luaL_optstring(L, 1, "/")`

Reply:
Ah cool - I was searching for "lua optional parameters" in the docs but couldn't find anything.
PS: Ah reading more, Lua's like JS in that all parameters are optional - my bad!